test/metabase/query_processor_test/timezones_test.clj

```diff
@@ -230,6 +242,16 @@
    (when (supports-datetime-with-zone-id?)
      {:datetime_tz_id (t/zoned-date-time "2019-11-01T00:23:18.331-07:00[America/Los_Angeles]")})))
 
+(deftest sql-datetime-timezone-handling-test
+  (mt/test-drivers (filter #(isa? driver/hierarchy % :sql) (conj (set-timezone-drivers) :oracle)) ; Oracle doesn't have a time type, so it's excluded from this test
```

Comment:
this comment makes no sense to me

Reply:
Also [hardcoding driver names in tests is a no-no](https://github.com/metabase/metabase/issues/26807), since it makes it really hard for third-party driver authors to run our test suite, and [I have a linter to disallow it in the works](https://github.com/metabase/metabase/pull/41384) and we're actively working on fixing existing tests that do it. 

Going forward we want to do something like 

https://github.com/metabase/metabase/blob/856b449/test/metabase/query_processor_test/timezones_test.clj#L250-L263

instead, so the right way to do this would be to come up with a namespaced feature flag for what you're trying to test and then add a default method and overrides as needed. More work but it makes things easier for third-party driver authors.

Reply:
I've removed this test for now since the oracle-specific behaviour is covered, I'll bring it back as part of [Add supports-base-type? multimethod for tests](https://github.com/metabase/metabase/pull/44581), and without any hard-coded driver names.